principles.md

```diff
@@ -0,0 +1,43 @@
+# Principles
+
+The goal of Vac is widespread adoption of the decentralized web. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+## I. Liberty
+
+We believe in the sovereignty of individuals. As a research organization that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms. This includes being coercion-resistant.
+
+## II. Censorship resistance
+
+We enable free flow of information. No content is under surveillance. We abide by the cryptoeconomic design principle of censorship resistance. Even stronger, we design agnostic infrastructures for information.
+
+## III. Security
+
+We don't compromise on security when building features. We use state-of-the-art technologies, and research new security methods and technologies to make strong security guarantees.
+
+## IV. Privacy
+
+Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as pseudo-anonymity. Additionally, we strive to provide the right of total anonymity.
```

Comment:
I'd remove  `pseudo-anonymity`. That leads to confusion, and actually sounds weak.
We could add `anonymity` as principle in the future, once there is more progress on the anonymity roadmap.

Reply:
1. This is not something we change haphazardly. The principles have a long history and change meaning of them like this shouldn't be done lightly. See https://our.status.im/our-principles/ and https://discuss.status.im/t/principles-how-to-sign-them-cryptographically/293

(doesn't mean it isn't impossible, just that it should be a deliberate effort)

2. Strong disagree on removing psuedo-anymity. This is a huge differentiator for e.g. Status on top of Waku vs other chat protocols that use phone numbers or other form of PII.

Reply:
Imo, it would need more explanation. Some definition. Pseudoanonymity gives the same "anonymity properties" as pseudonymity but, without a fixed "pseudonym". This phrase leads to confusion, imo.

> This is not something we change haphazardly. The principles have a long history and change meaning of them like this shouldn't be done lightly.
:+1: 

Imo, pseudo-anonymity sounds week, without an extra explantion. And it is leads to [questions like this](https://www.reddit.com/r/statusim/comments/w4svwn/please_define_statuss_anonymity/) amony users.
I also assume, many are familiar with "anonymity" and "pseudonymity"... but not with "pseudo-anonymity".

> Pseudonymity is meaningful and a big differentiator and something we should lean into more.

Agreed. But as a user, I'd expect an explanation as to why it is good. And.... the text says "pseudo-anonymity" not "pseudonymity".

Reply:
Fair point re *pseudo-anonymity* vs *pseudonymity*, I honestly didn't see this as a distinction. To me they point to the same concept, but your reading makes it seem like there are more ways to interpret it. Is it common to talk about something being "pseudo/fake" anonymity as a distinction from "pseudonymity?"

Reply:
> Is it common to talk about something being "pseudo/fake" anonymity as a distinction from "pseudonymity?"

Not that I know of. The term is much less used compared to pseudonymity though.
The "pseudo/fake" is just one interpretation, and was the first that came to my mind.
It could be defined as a superset of pseudonymity, though.
Still, I'd expect many readers not to be familiar with the term.

Reply:
> right, which is something we can explain in write-ups and such!

Yes. That will help a lot, imo. I'll put more stress on that in future posts.

Reply:
I'd suggest we take principles related feedback and collect it somewhere so we can see what a "soft fork" would look like, as this involves "governance" and would likely be a larger discussion with other stakeholders in terms of how Vac relates to Status and Logos.